### DISCUSSION TITLE
No way to hide users

### DISCUSSION THREAD

**Discussion by timcole** [community/community#13144] [Other Feature Feedback, Questions, & Ideas]
There currently doesn't seem to be a way to hide a user from the new For You feed outside of outright blocking the user.
Just because you don't wanna see what a user keeps starring (that you don't follow) doesn't mean they deserve to get blocked. Currently, my For You is just spam of the same user who I don't follow starring repos.

**Comment by idann** [comment]
Just to add -- even blocking a user doesn't remove them from the feed. I now have an obnoxious scareware repo at the top of the feed, permanently.

**Comment by Artturin** [comment]
we have a bot in nixpkgs that makes update PRs and i'd like to not see it making PRs
![image](https://user-images.githubusercontent.com/56650223/161712134-a7bf4b00-c6ce-4535-8694-8951e0498f13.png)

**Comment by ap--** [comment]
Same here.

My "for you" is now filled with news from the conda-forge bot.

![Screenshot 2022-04-05 at 13 35 40](https://user-images.githubusercontent.com/1463443/161745490-f8932cb3-8eef-449f-9be6-b9c0ba5eba65.png)

**Reply by sgbaird** [reply]
`regro-cf-autotick-bot` is everywhere on "For You" now. `conda-forge` has 16k feedstock repositories and counting, so as soon as you become a feedstock maintainer then:
![image](https://user-images.githubusercontent.com/45469701/171970196-5cfd108a-db81-4870-907e-d4c62f84069e.png)

**Comment by danielfdickinson** [comment]
I have a users coming up on 'For You' from projects I worked on years ago and where I no longer participate. The '…' on the top right shows 'You're seeing this because you collaborated with \<user>' but offers no way short of blocking the user to stop seeing that user's activity. Similarly with other users from the same GitHub organization, along with recommendation to add users from that org and/or collaborators with that org.

There really needs to be a way to tell the thing, "I'm not interested" and have it stick, without blocking users (I don't have anything against the users, I'm just not involved with that community these days).

**Comment by netuoso** [comment]
This is really a distracting nuisance that would be nice if we could silence. Can we please get some feedback on this issue? Honestly this could be used as a way to spread malware to unsuspecting users that don't even realize they aren't following the users they are seeing updates about.

**Comment by sgbaird** [comment]
Also, this post probably isn't getting as much visibility as it would normally because the "send feedback" links are broken #17895

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by danielfdickinson** [reply]
It doesn't solve the problem I mentioned where some 'For You' is based on folks I collaborated with years ago. This filter, while a good start, doesn't solve all the problems (although it does allow to completely turn off the feed, which is useful).

It filters by the wrong thing; it doesn't filter by the 'source' of the type of activity, it filters by the the activity itself (e.g. starring, follows, etc).

**Reply by jfullstackdev** [reply]
while waiting for this to be implemented in the future, you can simply block those users 

try ignoring the repo too, 

![image](https://user-images.githubusercontent.com/47092464/182068587-1287e387-38bb-4ff4-a18f-510ba6a81006.png)

**Reply by sumnerevans** [reply]
I can't afford to block the user because the bot plays an important part in my workflow by assigning me relevant issues in NixOS/nixpkgs, but I see all of those in notifications, and I don't need to know about every single PR that it creates.

**Comment by siminapasat** [comment]
We're working on adding the ability to dismiss users (hide them from the feed now and in the future) and should have an update very soon 🤞

**Reply by jfullstackdev** [reply]
yup, I think, that's one of the most requested, without blocking the user entirely

**Reply by drewboswell** [reply]
Yes please the bots and general automated integrations make notifications and feeds useless. I've stopped using them as it's a wall of renovate, policy-bot, bulldozer-bot, dependabot, github-actions commits etc.

Make notification useful again please.

**Reply by tarampampam** [reply]
> should have an update very soon

7mo ago - any news? 😊

**Reply by PathogenDavid** [reply]
It'd be great if you added this for repositories too. One thing I like from the new feed is passively seeing releases from projects I've starred, but a few projects have their CI deploying releases and they end up constantly polluting my feed:

![image](https://github.com/community/community/assets/278957/225e8e84-29a3-4574-8497-a9e2fd44b44e)

I'd rather not unstar these repositories as I do want to be able to find them again, but I don't need to be reminded that they exist every single day. (Being able to mute pre-releases might also be valuable.)

**Comment by yunnysunny** [comment]
A concrete case that is still causing significant noise: commits authored by `github-actions[bot]` frequently fill the GitHub home Feed.

I still want to see human commits and other relevant activity from these repositories. Unfollowing or unstarring the repositories is therefore not a useful workaround, and hiding all push/commit events is too coarse because it also removes meaningful human activity.

Please add a Feed-level filter that can exclude:

- a specific actor, such as `github-actions[bot]`; and/or
- actors by account type, such as **Bot**.

Ideally, this preference would affect only the Feed and would not block the bot, alter repository access, or suppress unrelated notifications. This would substantially improve the signal-to-noise ratio for repositories that use Actions for automated version bumps, generated files, deployment commits, or other routine maintenance.
